### PR TITLE
Adding kafka landing page

### PR DESCRIPTION
**Pull request by johnsimons**

### CODE REVIEW

File: menu/menu.yaml [hunk 1]
```diff
@@ -875,6 +875,10 @@
       Articles:
       - Url: transports/upgrades/msmq-1to2
         Title: Version 1 to 2
+  - Title: Kafka
+    Articles:
+    - Url: transports/kafka
+      Title: Kafka transport
```
Comment (lailabougria) [inline menu/menu.yaml:881]: As we discussed if we follow through on this PR, we shouldn't place this under transports as it sets the wrong expectations.

File: transports/kafka/index.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Kafka Transport
+summary: A Kafka based transport for NServiceBus
```
Comment (lailabougria) [inline transports/kafka/index.md:3]:
```suggestion
title: Kafka integration
summary: Kafka integration into the Particular Service Platform
```

File: transports/kafka/index.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+reviewed: 2024-01-24
+---
+
+![Kafka logo](https://apache.org/logos/res/kafka/kafka_highres.png)
+
+We are considering creating a Kafka based transport for NServiceBus, but we need your help to better shape this transport.
```
Comment (lailabougria) [inline transports/kafka/index.md:9]:
```suggestion
We are gauging interest and gathering insights into the relevant use cases for Kafka integration in the Particular Service Platform and we need your help!
```

File: transports/kafka/index.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+Please complete [this survey](https://2wq02jeoug5.typeform.com/to/bpjaVSDF) to help us design the best Kafka transport.
```
Comment (lailabougria) [inline transports/kafka/index.md:10]:
```suggestion
Please complete [this survey](https://2wq02jeoug5.typeform.com/to/bpjaVSDF) to help us understand your use cases to build the best possible integration with Kafka.
```

**Comment by lailabougria** [comment]
I reviewed it as promised, but I think we should close this PR in favor of https://github.com/Particular/docs.particular.net/pull/6444

File: menu/menu.yaml (hunk 1 above)
Comment (bbrandt) [reply menu/menu.yaml:881]: There may be many that already have some sort of expectation, [based on other libraries](https://masstransit.io/documentation/transports/kafka).

**Pull request closed without merging**